modules/nw-ovn-kubernetes-rollback.adoc

```diff
@@ -31,41 +31,70 @@ endif::[]
 
 . Stop all of the machine configuration pools managed by the Machine Config Operator (MCO):
 
-** Stop the master configuration pool:
+** Stop the `master` configuration pool:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
 $ oc patch MachineConfigPool master --type='merge' --patch \
   '{ "spec": { "paused": true } }'
 ----
 
-** Stop the worker machine configuration pool:
+** Stop the `worker` machine configuration pool:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
 $ oc patch MachineConfigPool worker --type='merge' --patch \
   '{ "spec":{ "paused": true } }'
 ----
 
 . To prepare for the migration, set the migration field to `null` by entering the following command:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
 $ oc patch Network.operator.openshift.io cluster --type='merge' \
   --patch '{ "spec": { "migration": null } }'
 ----
 
-. To start the migration, set the network plugin back to OpenShift SDN by entering the following commands:
+. Apply the patch to the `Network.operator.openshift.io` object to set the network plugin back to OpenShift SDN by entering the following command:
 +
-[source,terminal]
+[source,terminal,subs="+quotes"]
 ----
 $ oc patch Network.operator.openshift.io cluster --type='merge' \
   --patch '{ "spec": { "migration": { "networkType": "OpenShiftSDN" } } }'
+----
+
+. Confirm that the `Network.operator.openshift.io cluster` object's network plugin is `OpenShiftSDN` by entering the following command. If the command output does not show `OpenShiftSDN`, wait a few seconds for the patch operation to complete.
++
+[source,terminal,subs="+quotes"]
+----
+$ oc get network.config cluster -o jsonpath='{.status.migration.networkType}'
+----
++
+[IMPORTANT]
+====
+If you applied the patch to the `Network.config.openshift.io` object before the patch operation finalizes on the `Network.operator.openshift.io` object, the Cluster Network Operator enters into a degradation state. You can revert this configuration by entering the following command:
```

Comment:
Users may also need to remove the ConfigMap `applied-cluster` in namespace `openshift-network-operator`.

Reply:
Thanks, @pliurh . I do not see `applied-cluster` used in our customer docs so is still an offical namespace entry? All ConfigMaps or just one? How about:

>  To revert this configuration, you can enter the following command and, if necessary, remove `applied-cluster` from the `openshift-network-operator` namespace in the `ConfigMaps` in your OVN-Kubernetes project.

Reply:
`applied-cluster` is a ConfigMap object used by the cluster network operator. In most cases, users don't need to know about it. The sentence should be:
> remove the `applied-cluster` ConfigMap object from the openshift-network-operator namespace

Reply:
Thanks, updated.